src/components/CardModal/CardModalActions/AttachmentMenu/AttachmentMenu.tsx

```diff
@@ -21,13 +21,15 @@ const AttachmentMenu = ({ update }: AttachmentMenuProps) => {
 
     const files = e.target.files;
     const upload = [];
+    const setFilePromises = [];
     if (files) {
       for (let i = 0; i < files.length; i++) {
         const id = uuidv4();
         const date = JSON.stringify(new Date());
-        await localforage.setItem(id, files[i]);
+        setFilePromises.push(localforage.setItem(id, files[i]));
         upload.push({ id, date, name: files[i].name });
       }
+      await Promise.all(setFilePromises);
```

Comment:
I've changed this slightly to make testing easier. Granted, this isn't a best practice but when testing this, it's hard to check the date is what we expect as the await inside the loop causes the date to be different than the mock. I couldn't find a solution in jest for this (though I'm sure there is one), so in the interest of saving time and progressing, I've changed this logic slightly to gather all the setItem promises and await all of them once the for loop is done.